sql/core/src/main/scala/org/apache/spark/sql/sources/ddl.scala

```diff
@@ -231,6 +241,32 @@ private [sql] case class CreateTempTableUsing(
   }
 }
 
+private[sql] case class DropTable(
+    tableName: String,
+    isExists: Boolean,
+    temporary: Boolean) extends Command
```

Comment:
hi, @liancheng, I think we do need this logical `DropTable`. 
Since all parser should go first ddlParser, if not get plan, then it will try dialect parser. 
If I remove this `logical drop table`, when execute `drop table xxx`, it will always use `DropTableCommand` in `SQLContext`.  Sorry if I'm wrong.
In HiveContext.
>>
 else if (conf.dialect == "hiveql") {
      new SchemaRDD(this, ddlParser(sqlText, false).getOrElse(HiveQl.parseSql(substituted)))